android/src/main/java/com/aerobotics/DjiMobile/FlightControllerWrapper.java

```diff
@@ -457,6 +457,18 @@ public void stopOnboardSDKDeviceDataListener(final Promise promise) {
     promise.resolve(null);
   }
 
+  @ReactMethod
+  public void setPowerSupplyPortEnabled(Boolean enabled, Promise promise) {
+    Aircraft product = ((Aircraft)DJISDKManager.getInstance().getProduct());
+    if (product != null) {
+      final FlightController flightController = product.getFlightController();
+      flightController.setPowerSupplyPortEnabled(enabled);
+      promise.resolve(null);
```

Comment:
I've found that it's better to do `promise.resolve(true)`, some JS logic requires comparing a resolved value of the promise, so resolving `true` makes it easier to do! (I know resolving `null` is what happens throughout the library, there's lots of learning from this work!)

Reply:
Thanks @arosendorff for the pointer. I've updated 👍